index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Exercício 03 - Orion Bootcamp</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      background-color: #4B0082;
+      color: #F5F5F5;
+      margin: 0;
+      padding: 0;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+
+    h1 {
+      color: #FF6F61;
+      margin-top: 20px;
+    }
+
+    p {
+      max-width: 600px;
+      text-align: center;
+      color: #E6E6FA;
+    }
+
+    div {
+      background-color: #6A5ACD;
+      padding: 20px;
+      margin: 20px;
+      border-radius: 10px;
+      box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
+      max-width: 600px;
+      width: 100%;
+    }
+
+    label {
+      font-weight: bold;
+      display: block;
+      margin-bottom: 5px;
+    }
+
+    input[type="text"],
+    select {
+      padding: 10px;
+      border: 1px solid #6A5ACD;
+      border-radius: 5px;
+      width: calc(100% - 22px);
+      margin-bottom: 10px;
+    }
+
+    button {
+      background-color: #FF6F61;
+      color: white;
+      padding: 10px 20px;
+      border: none;
+      border-radius: 5px;
+      cursor: pointer;
+      font-weight: bold;
+    }
+
+    button:hover {
+      background-color: #FF836F;
+    }
+
+    #resultadoVogais,
+    #resultadoOperacao {
+      margin-top: 15px;
+      color: #E6E6FA;
+      background-color: #4B0082;
+      padding: 10px;
+      border-radius: 5px;
+      font-size: 1.2em;
+      text-align: center;
+      box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
+    }
+
+    .hidden {
+      display: none;
+    }
+
+    #historicoCommits {
+      background-color: #6A5ACD;
+      padding: 20px;
+      margin: 20px;
+      border-radius: 10px;
+      box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
+      max-width: 600px;
+      width: 100%;
+    }
+
+    #imgHistorico {
+      max-width: 100%;
+      border-radius: 10px;
+      box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
+    }
+  </style>
+</head>
+
+<body>
+  <h1>Orion Bootcamp</h1>
+
+  <div>
+    <h2>Contagem de Vogais</h2>
+    <h4>Digite uma palavra para descobrir quantas vogais ela possui.</h4>
+    <label for="palavraInput">Palavra:</label>
```

Comment:
Uma dica de usabilidade, quando for usar input que precisa de um botão pra finalizar é legal usar o form do html com um botão do tipo submit, além de melhorar a semântica das tags e sinalizar para os navegadores que isso é um form, ao digitar no input e apertar enter ele já vai acionar o botão automaticamente, teria que mudar um pouco o código pra funcionar, pois você iria precisar usar o eventListener no submit do form, mas o html ficaria assim:

```
<form>
      <label for="palavraInput">Palavra:</label>
      <input type="text" id="palavraInput" placeholder="Digite uma palavra"
        aria-label="Digite uma palavra para contagem de vogais">
      <button type="submit" id="btnContarVogais">Contar Vogais</button>
    </form>
```